msteams-platform/messaging-extensions/design/messaging-extension-design.md

```diff
@@ -220,15 +220,15 @@ Use these recommendations to create a quality app experience.
 
 :::row:::
    :::column span="":::
-:::image type="content" source="../../assets/images/messaging-extension/setup-do.png" alt-text="Example on setup and general usage.":::
+:::image type="content" source="../../assets/images/messaging-extension/setup-do.png" alt-text="Example of setup and general usage.":::
```

Comment:
Repeat feedback: think about what does the image path `../../assests/images` mean.